examples/deepcopy-gen/main.go

```diff
@@ -22,7 +22,10 @@ limitations under the License.
 // generating new ones.
 //
 // If a "func (in *T) postDeepCopy(out *T)" func is predefined on an object by the
```

Comment:
Why can't this logic just check for and call the T.DeepCopy()?  If the use case is to not generate code and then call custom code, that is exactly what defining DeepCopy does.